src/scores/output.py

```diff
@@ -0,0 +1,106 @@
+"""Creating the final Excel output of the project."""
+
+from datetime import date
+
+from pandas import DataFrame, ExcelWriter
+from xlsxwriter import Workbook
+from xlsxwriter.format import Format
+from xlsxwriter.worksheet import Worksheet
+
+from src.config import tables
+from src.utils import read_csv
+
+
+def format_between(format: Format, min: float, max: float):
```

Comment:
Missing return type

Reply:
Python automatically infers `dict[str, Unknown]`. This could be improved upon like with `CheckReturnList`, but that one is currently defined as `list[dict[str, Any]]`. If we add something, it should be a bit more helpful than that.

<img width="643" alt="Screenshot 2024-09-01 at 12 31 34" src="https://github.com/user-attachments/assets/c4f2f8b0-3fc0-44ed-8b5a-bcde5a73e483">